devicon.json

```
@@ -156,6 +157,24 @@
             }
         ]
     },
+    {
+        "name": "appwrite",
+        "tags": ["cloud", "platform", "server"],
+        "versions": {
+          "svg": [
+            "original",
+            "original-wordmark",
+            "plain",
+            "plain-wordmark"
+          ],
+          "font": [
+            "original",
+            "original-wordmark",
+            "plain",
+            "plain-wordmark"
+          ]
+        }
```

Adding the `color` for colored versions and aliases to not have the same icon twice in the font (only in css).

```suggestion
        },
        "color": "#f02e65",
        "aliases": [
            {
                "base": "plain",
                "alias": "original"
            },
            {
                "base": "plain-wordmark",
                "alias": "original-wordmark"
            }
        ]
```